app/assets/src/components/common/MetadataManualInput.jsx

```diff
@@ -249,6 +249,7 @@ class MetadataManualInput extends React.Component {
   isHostGenomeIdValidForField = (hostGenomeId, field) =>
     // Special-case 'Host Genome' (the field that lets you change the Host Genome)
     field === "Host Genome" ||
+    this.props.projectMetadataFields[field].is_required ||
```

Comment:
Is there any chance of field not being in `this.props.projectMetadataFields`?
If so, you can always use the `get([field, "is_required"]` as below. Sometimes is good to do it just to be on the safe side.

Reply:
Good point, to be safe.